src/libraries/System.Drawing.Common/src/Resources/Strings.resx

```diff
@@ -353,6 +353,9 @@
   <data name="PlatformNotSupported_Drawing" xml:space="preserve">
     <value>System.Drawing is not supported on this platform.</value>
   </data>
+  <data name="PlatformNotSupported_Unix" xml:space="preserve">
+    <value>System.Drawing.Common is not supported on non-Windows platforms. See https://aka.ms/systemdrawingnonwindows for more information.</value>
```

Comment:
Do we need the existing  string above as well?

Reply:
Yes. That is used for platforms that we can't enable drawing on, for example Browser, iOS, etc. On .NET 7 we will be able to merge these two.